data/corsika/corsika_parameters.yml

```diff
@@ -0,0 +1,105 @@
+USER_PARAMETERS:
+  RUNNR:
+    len: 1
+    names: ['RUNNUMBER', 'NRUN']
+    default: 1
+  EVTNR:
+    len: 1
+    names: ['EVENTNR', 'EVENTNUMBER']
+    default: 1
+  NSHOW:
+    len: 1
+    names: ['NSHOWERS']
+  PRMPAR:
+    len: 1
+    names: ['PRIMARY']
+  ERANGE:
+    len: 2
+    names: ['ENRANGE', 'ENERGYRANGE']
+    unit: [!astropy.units.Unit {unit: GeV}, !astropy.units.Unit {unit: GeV}]
+  ESLOPE:
+    len: 1
+    names: ['ENSLOPE', 'ENERGYSLOPE']
+  THETAP:
+    len: 2
+    names: ['THETA', 'ZENITH']
+    unit: [!astropy.units.Unit {unit: deg}, !astropy.units.Unit {unit: deg}]
+  PHIP:
+    len: 2
+    names: ['PHI', 'AZIMUTH']
+    unit: [!astropy.units.Unit {unit: deg}, !astropy.units.Unit {unit: deg}]
+  VIEWCONE:
+    len: 2
+    names: ['CONE']
+    unit: [!astropy.units.Unit {unit: deg}, !astropy.units.Unit {unit: deg}]
+  CSCAT:
+    len: 3
+    names: ['SCAT']
+    unit: [null, !astropy.units.Unit {unit: cm}, null]
+
+PRIMARIES:
+  GAMMA:
+    number: 1
+    names: ['PHOTON']
+  ELECTRON:
+    number: 3
+    names: ['ELECTRONS']
+  PROTON:
+    number: 14
+    names: ['PROTONS']
+  HELIUM:
+    number: 402
+    names: []
```

Comment:
Would add here for consistency 'HELIUM"

Reply:
But 'HELIUM' is already the main name. The other particles had the plural in the names.
BTW, this list is incomplete. I will complete it now and push it before merging.

Reply:
Ahh - didn't see that it is the plural for the others. All good.